src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java

```diff
@@ -123,4 +126,19 @@ public void clickOnRegisterLink(){
   public String getLoginHeaderText(){
     return loginHeader.getText();
   }
+
+  public void typePassword(String password) { passwordField.sendKeys(password); }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)